make/template/inspircd

```diff
@@ -88,12 +88,11 @@ if (!defined($sub))
 {
 	print STDERR "Invalid command or none given.\n";
 	cmd_help();
-	exit 1;
+	exit 3;	# Unimplemented feature
```

Comment:
Is there any need for the comments? You can just put a link to the spec at the top somewhere.